README.md

```diff
@@ -145,7 +184,9 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+See Sentence.cpp
+(For some reason when running in the genuse servers at smu the last test case would bug out,
+but running on my machine it always returns false, just FYI).
```

Comment:
# 12

# +5 bonus

You used a map! And it was keyed on the state/event pair! And you even did it using the ridiculous C++ syntax!

You made me very happy: almost no one used a table, and you're trhe only person (so far) to use a map.

Thank you.